Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -341,6 +340,7 @@
 
         <ContextMenuAction FunctionName="LandingPad_Test_ContextMenuAction" Text="Lua Test Deconstruct 1" RequiereCharacterSelected="false"/>
         <ContextMenuAction FunctionName="LandingPad_Test_ContextMenuAction" Text="Lua Test Deconstruct 2" RequiereCharacterSelected="true" />
```

Comment:
Now that we have proper context actions, you should remove the two test actions.